autopush/settings.py

```diff
@@ -204,18 +206,35 @@ def message(self, value):
         """Setter to set the current message table"""
         self.message_tables[self.current_msg_month] = value
 
+    def add_tomorrow(self, today, this_month_table):
+        """check to see if tomorrow is a new month, in case we're a node that
+        has a slow clock."""
+        tomorrow = get_rotating_message_table(
+            self._message_prefix, 0, today + datetime.timedelta(days=1))
+        if tomorrow.table_name != this_month_table:
+            self.message_tables[tomorrow.table_name] = Message(
```

Comment:
update_rotating_tables is called once a minute. We should check to ensure tomorrow's table isn't already in there before making it again.